msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -82,26 +77,30 @@ Register you application at the registration portal for the Azure AD v1.0 endpoi
     * openid
     * profile
 
+> [!NOTE]
+> * ¹ If your AAD app is registered in the _same_ tenant that you're making an authentication request from in Teams, then the user will not be asked to consent and instead will be granted an access token right away. Users only need to consent to these permissions if the AAD app is registered in a different tenant.
+> * ² If you get an error saying that the domain is already owned but you own it, follow the procedure at [Quickstart: Add a custom domain name to Azure Active Directory](/azure/active-directory/fundamentals/add-custom-domain) to register it, and then repeat this step. (This error can also occur if you are not signed in with credentials of an admin in the Office 365 tenancy).
+
+
 ### 2. Update your Microsoft Teams application manifest
 
 Add new properties to your Microsoft Teams manifest:
 
 * **WebApplicationInfo** - The parent of the following elements.
-* **Id** - The client ID of the application. This is an application ID that you obtain as part of registering the application with Azure AD 1.0 endpoint.
-* **Resource** - The domain and subdomain of your application. This is the same URI (including the `api://` protocol) that you used when registering the app in AAD. The domain part of this URI should match the domain, including any subdomains, used in the URLs in the section of your Teams application manifest.
+* **id** - The client ID of the application. This is an application ID that you obtained as part of registering the application with Azure AD.
+* **resource** - The domain and subdomain of your application. This is the same URI (including the `api://` protocol) that you registered when creating your `scope` in step #4 above. You shouldn't include the `access_as_user` path in your resource. The domain part of this URI should match the domain, including any subdomains, used in the URLs of your Teams application manifest.
 
 ```json
 "webApplicationInfo": {
-  "id": "<application_GUID here>",
-  "resource": "<web_API resource here>"
+  "id": "c6c1f32b-5e55-4997-881a-753cc1d563b7",
+  "resource": "api://subdomain.example.com/c6c1f32b-5e55-4997-881a-753cc1d563b7"
 }
 ```
 
 Notes:
 
 * The resource for an AAD app will usually just be the root of its site URL and the appID (e.g. `api://subdomain.example.com/6789/c6c1f32b-5e55-4997-881a-753cc1d563b7`). We also use this value to ensure your request is coming from the same domain. Therefor make sure that your `contentURL` for your tab uses the same domains as your resource property.
```

Comment:
It would be helpful to mention that `6789` is the port number in 
`api://subdomain.example.com/6789/c6c1f32b-5e55-4997-881a-753cc1d563b7`  
@aosolis : Please correct me if I am wrong about the port.

We also seem to be using c6c1f32b-5e55-4997-881a-753cc1d563b7 as the sample id everywhere. Wouldn't it be better to have a placeholder ?

Reply:
@lokimeyburg what is the 6789 here? Is the port number required, or just the domain?

Reply:
Got rid of the `6789`. That was carry-over from the port. Removed it for clarity. I also switched the `app_id` to just zeros. Wasn't a fan of using {appId} placeholders.